exonum-java-binding/core/rust/src/proto/mod.rs

```diff
@@ -17,6 +17,4 @@
 #![allow(bare_trait_objects)]
 #![allow(renamed_and_removed_lints)]
 
-pub use self::service_runtime::*;
```

Comment:
The whole module is unusable now, it seems. I don't want to delete it, because it might be useful in the future.